examples/image_captioning/image_captioning.py

```diff
@@ -0,0 +1,115 @@
+import ludwig
+import tensorflow as tf
+import os
+import pandas as pd
+import numpy as np
+from ludwig.api import LudwigModel
+import logging
+
+################### DOWNLOAD AND PREPROCESS DATA ###################
+# Download data
+image_folder = os.path.abspath('.') + '/Flicker8k_Dataset/'
```

Comment:
Can you add this as a dataset to `ludwig.datasets`?  That way it can be reused even outside of this example.

Reply:
@tgaddair just made a PR for this at #1053